__tests__/__system__/cli/execute/cli.db2.execute.sql.system.test.ts

```diff
@@ -73,6 +83,23 @@ describe("db2 execute sql command", () => {
         expect(response.status).toBe(0);
     });
 
+    it("should be able to execute SQL statements using passed options not profile", () => {
+        TEST_ENV.tempProfiles = null;
```

Comment:
I don't think this will prevent profiles from being used on commands. You would have to create a new test environment to test  having no profiles. Setting this to null will just prevent the profiles from being deleted when you call TestEnvironment.cleanUp. Same comment for other tests where you did this